### DISCUSSION TITLE
Motion Capture Recommendations for 10ish Crazyflies

### DISCUSSION THREAD

**Discussion by phillips0616** [USC-ACTLab/crazyswarm#615] [Q&A] [answered]
When flying 10 or so crazyflies are unique or duplicate marker configurations more commonly used? I use optitrack motion capture and have tried both. I seem to get more stable flight when using libobjecttracker with duplicate arrangements, but wasn't sure how it scales as you increase the number of crazyflies. 

In case it affects the answer.. I am almost exclusively controlling them in low-level mode using cmdVelocityWorld() and am working on a custom collision avoidance module, so I would expect them to be in close proximity a lot of the time.

**Comment by jpreiss** [comment, marked as answer]
It depends on the quality of the mocap system's tracking. With Vicon Tracker I have seen instability even with a relatively small number of unique configurations. E.g. if you take the default marker configuration here:

![image](https://user-images.githubusercontent.com/1606009/158648669-d9aa578a-c5d8-4664-a358-d8bae534e490.png)

and put the top marker at each of the 4 corners of the battery retainer, you get 4 unique configurations. Those configurations should be easily distinguished at Vicon's <1mm resolution. But in practice our Vicon setup gives unstable tracking with swaps sometimes.

(There could be some suboptimal choice in our Vicon configuration though. There are so many parameters to tune.)

So with 10 standard CFs I would probably go libobjecttracker. With a frame or a bigger quadrotor you could make maker arrangements that are more different.
